contracts/modules/cover/Cover.sol

```diff
@@ -473,6 +473,10 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     coverId = coverNFT.mint(newOwner);
     _coverData[coverId] = CoverData(productId.toUint24(), coverAsset.toUint8(), 0 /* amountPaidOut */);
 
+    uint burnedSegmentBucketId = Math.divCeil((start + period), BUCKET_SIZE);
+    activeCoverExpirationBuckets[coverAsset][burnedSegmentBucketId] += amount;
+    activeCover[coverAsset].totalActiveCoverInAsset += amount.toUint192();
```

Comment:
do we need to adjust the other field of activeCover as well here?

Reply:
`lastUpdateBucketId` should only be updated when expirations are processed. This happens on cover buy, in theory we could have them on migration, but this is optional, I'd leave them out for simplicity.